### DISCUSSION TITLE
适配即刻的文章页与转发页

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2771] [站点适配]
> 如果喜欢在 Web 端使用黄即的话，可以试试简悦的这几个适配规则。

![image](https://user-images.githubusercontent.com/81074/211244093-f229ba44-8c6d-46d5-94c2-cd0871cfcf9a.png)

# 功能

- 去掉干扰元素，直接获取正文。

- 进入阅读模式后，可以方便的将你喜欢的内容导入到：Notion / Obsidian / Logseq 等简悦支持的服务。

# 特色

- 标题使用 ID 和时间戳，方便保存时留档

- 转发页显示转发内容

- 显示当前内容所在的圈子

<img width="1820" alt="image" src="https://user-images.githubusercontent.com/81074/211243867-9a2e5019-031e-4b08-924e-6a488bd710c4.png">

# 如何使用

> 无需任何设置，仅需安装下面三个站点规则。

1. 安装 [即刻-动态页面](https://simpread.ksria.cn/sites/details/W03SKM5Qe3) 的适配规则。

2. 安装 [即刻-转发页面](https://simpread.ksria.cn/sites/details/9AVc5snhdd) 的适配规则。

3. 安装 [即刻-Mobile 页面](https://simpread.ksria.cn/sites/details/ucpaKpybxj) 的适配规则。

# 加入排除列表（可选）

如果你不需要每次打开都自动进入阅读模式的话，将 `okjike.com` 加入排除列表，当有需要的时候再进入阅读模式。

<img width="1168" alt="image" src="https://user-images.githubusercontent.com/81074/211243764-43e3d9e4-0f85-46ab-9417-df2016533636.png">

# 效果

![image](https://user-images.githubusercontent.com/81074/211254187-152acc9a-9846-4e72-b09c-ef7170d4b731.png)

# 关联

- [适配了一组微博页面](https://github.com/Kenshin/simpread/discussions/3515)

- [适配 Twitter 推文页](https://github.com/Kenshin/simpread/discussions/2923)

- [适配 Thread Reader](https://github.com/Kenshin/simpread/discussions/2046)

- [适配 Typefully 内容页](https://github.com/Kenshin/simpread/discussions/3517)

# 引申

使用同步助手的自动同步无法安装适配规则的解决方案 https://github.com/Kenshin/simpread/discussions/4859

**Comment by xiang578** [comment]
安装 即刻-动态页面 的规则后，https://web.okjike.com/originalPost/602f1662a1add00018dc7faa 这个页面解析还是错误。

多了一张图片，标题也不对

![simpread-楚甜： 看了下这几天的粉丝增长，回想起之前老大跟我说的话：__老大说：__不是给你 P7 的职位，你就能做 P7 的事情；_不是给你带新人，你就能学会带新人；_不是给你机会，](https://user-images.githubusercontent.com/12624910/202911441-7d94d193-478c-400b-98f8-b0db278be02b.png)

**Reply by Kenshin** [reply]
嗨，新的适配规则已更新，请重新安装。
